src/components/Display.jsx

```diff
@@ -0,0 +1,17 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+
+// eslint-disable-next-line react/prefer-stateless-function
+const Display = ({ result }) => {
+  return (
+    <div className="result">
+      <h1>{result}</h1>
+    </div>
+  );
+};
+
+Display.propTypes = {
+  result: PropTypes.string.isRequired,
+};
```

Comment:
- [ ] Here we should have the same behavior. The result props should have the default value defined as '0' and it does not need to be required.